stable/cachet/values.yaml

```
@@ -24,6 +24,9 @@ ingress:
   host: chart-example.local
   tls: []
   path: /
+  # -- Set ingressClass for Ingress when needed
+  # ingressClassName: ingressClass
```

Make this change, rebase from master if required, then regenerate the docs and commit the changes. Then it should pass 🙂

```suggestion
  # ingress.ingressClassName -- Set ingressClass for Ingress when needed
  ingressClassName: ""
```